nuxeo-core-binarymanager-s3-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-s3-direct-upload/nuxeo-s3-direct-upload.html

```diff
@@ -127,11 +128,26 @@
       }.bind(this));
     };
 
+    S3Provider.prototype.hasAbort = function() {
```

Comment:
For us to take advantage of this on elements using `Nuxeo.UploaderBehavior`, we'd have to add to this behavior a respective method:
```
 hasAbort() {
        return this._instance.hasAbort();
}
``` 
However, I don't think we need this yet if we take https://github.com/nuxeo/nuxeo-web-ui/pull/438#discussion_r246051800 into account.